contracts/LifCrowdsale.sol

```diff
@@ -346,18 +357,56 @@ contract LifCrowdsale is Ownable, Pausable {
       foundationWallet.transfer(foundationBalanceCapWei);
 
       // TODO: create the MVM with a start block that equals one month after crowdsale ends
-      LifMarketValidationMechanism newMVM = new LifMarketValidationMechanism(
+      MVM = new LifMarketValidationMechanism(
         address(token), block.timestamp.add(10), 30 days, MVMPeriods, foundationWallet
       );
-      newMVM.fund.value(mmFundBalance)();
-      newMVM.transferOwnership(foundationWallet);
 
-      MVM = address(newMVM);
+      mintExtraTokens(uint256(MVMPeriods));
+
+      MVM.fund.value(mmFundBalance)();
+      MVM.transferOwnership(foundationWallet);
 
     }
   }
 
   /**
+     @dev Internal. Distribute the rest of the tokens among founders and advisors
+   */
+  function mintExtraTokens(uint256 foundationMonthsStart) internal {
+    // calculate how much tokens will the founders, foundation and advisors will receive
+    uint256 foundersTokens = token.totalSupply().mul(128).div(1000);
+    uint256 foundationTokens = token.totalSupply().mul(50).div(1000);
+    uint256 advisorsTokens = token.totalSupply().mul(72).div(1000);
+
+    // create the vested payment schedule for the founders
+    foundersVestedPayment = new VestedPayment(
+      block.timestamp, 30 days, 48, 12, foundersTokens, token
+    );
+    token.mint(foundersVestedPayment, foundersTokens);
+    foundersVestedPayment.transferOwnership(foundationWallet);
+
+    // create the vested payment schedule for the foundation
+    uint256 foundationPaymentStart = foundationMonthsStart.mul(604800);
```

Comment:
604800 is a week, I think we are talking about months here. Maybe it's better to use `foundationMonthsStart.mul(1 month)` (or `30 days` instead of `1 month`)

Reply:
:+1: